Src/BootCamp.Chapter/Program.cs

```diff
@@ -7,26 +7,31 @@ class Program
         static void Main(string[] args)
         {
             //Homework 1 and 2 
-            //1
-            string name, surename;
-            int age;
-            double weight, height, heightMeters, bmi;
-
+          
+            string name;
```

Comment:
You don't have to declare variables in separate lines. It's better for readaiblitiy if you assign and declare it at the same line, if the assignment is simple, which in this case  it is.